Sources/Sentry/Transactions/ISentrySpan.h

```diff
@@ -0,0 +1,17 @@
+//
+//  ISpan.h
+//  Sentry
+//
+//  Created by Dhiogo Brustolin on 05/01/21.
+//  Copyright © 2021 Sentry. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol ISentrySpan <NSObject>
```

Comment:
@philipphofmann what will we name the actual class? On Android we chose to use `I` as a convention for such cases where the interface matches the implementation name .. Actually I think we just endded up prefixing all with `I` at the end, similar to C#

Reply:
I know the `I` pattern, and personally, I'm not a huge fan of it. I think you should avoid it whenever possible. Name the interface after the concept and the actual implementation should tell more about the implementation itself. For example, in Java, you have `List` and `ArrayList`, `CopyOnWriteArrayList`, ...
The `I` pattern is more common in Java and C#, but I don't think it is common in Objective-C or Swift, or am I wrong @brustolin?

Reply:
I guess they can have the same name, but the file has a different extension, using `@interface` and `@ implementation `, if this holds true, I'd avoid the `I` as well

Reply:
Thats correct, the I pattern is not common in Objc. Since I was testing I just copied from java implementation until I realized I didn't need it to capture a transaction, so, some classes are not being used yet.

Open to new name options.

Apple naming suggestion: 
"Most protocols group related methods that aren’t associated with any class in particular. This type of protocol should be named so that the protocol won’t be confused with a class. A common convention is to use a gerund (“...ing”) form"

"Some protocols group a number of unrelated methods (rather than create several separate small protocols). These protocols tend to be associated with a class that is the principal expression of the protocol. In these cases, the convention is to give the protocol the same name as the class."

I guess we could go for the same name?

Reply:
sounds good to me